examples/cpp/raw_mandelbrot.cpp

```diff
@@ -0,0 +1,175 @@
+#include <stdlib.h>
+#include <dlfcn.h>
+#include <stdio.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <new>
+
+#include "pf.h"
+
+#include "model/colormap.h"
+#include "model/image.h"
+#include "model/enums.h"
+#include "model/imagewriter.h"
+
+#include "model/fractfunc.h"
+#include "model/vectors.h"
+
+
+int main() {
+    // initial setup: load fract4_stdlib globally so the loaded formula has access to it
+    void *fract_stdlib_handle = dlopen("./fract_stdlib.so", RTLD_GLOBAL | RTLD_NOW);
+    if (NULL == fract_stdlib_handle) {
```

Comment:
prefer `nullptr`. NULL is a C macro, nullptr is a considerably more robust way of expressing this.